samples/net/echo_client/boards/qemu_cortext_m3.conf

```diff
@@ -1,7 +0,0 @@
-# The addresses are selected so that qemu<->qemu connectivity works ok.
```

Comment:
Note sure why this was removed. Just keep this file, and put the stellaris specific stuff into overlay-stellaris.conf file.